mava/components/updating/checkpointer.py

```diff
@@ -31,6 +32,7 @@
 @dataclass
 class CheckpointerConfig:
     checkpoint_minute_interval: float = 5
+    restore_best_net: str = "None"
```

Comment:
Would it make sense to have a boolean here? True to restore and False to not restore from best checkpoints.

Reply:
Same as above, we have multiple best checkpointed network each belong to a specific metric so we need config to mention if someone want to have win rate best net or reward best net..